thinc/shims/pytorch.py

```diff
@@ -40,6 +47,8 @@ def __init__(
         mixed_precision: bool = False,
         grad_scaler: Optional[PyTorchGradScaler] = None,
         device: Optional["torch.device"] = None,
+        serialize_model_config: Optional[Callable[[Any], Dict[str, Any]]] = None,
+        deserialize_model_from_config: Optional[Callable[[Dict[str, Any]], Any]] = None,
```

Comment:
I gave this some more thought and realized that it may be better to give the serialization functions full control of the serialization of the Torch model? So the signatures would be:

```suggestion
        serialize_model_config: Optional[Callable[[Any], bytes]] = None,
        deserialize_model_from_config: Optional[Callable[[bytes], Any]] = None,
```

In this way, the caller could also apply more advanced transformations, such as removing certain parameters, etc.